src/EventPage/EventPage.js

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { View, Text, Image, StyleSheet, FlatList } from 'react-native';
```

Comment:
once we move the styles out of the file, we can remove the dependency on `StyleSheet`